src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/product-part-list.tpl.html

```diff
@@ -0,0 +1,89 @@
+<div class="blade-static">
+    <div class="form-group">
+        <div class="form-input __search">
+            <input ng-model="filter.keyword"
+                   placeholder="{{'platform.placeholders.search-keyword' | translate}}"
+                   ng-keyup="$event.which === 13 && filter.criteriaChanged()" />
+            <button class="btn __other"
+                    style="position: relative; right: 45px;">
+                <i class="btn-ico fa fa-times-circle"
+                   title="Clear"
+                   ng-click="filter.keyword=null;filter.criteriaChanged()"
```

Comment:
Better move it to a separate method of ctrl.

Reply:
Fixed